src/calculation/MeasureReportBuilder.ts

```diff
@@ -497,27 +629,12 @@ export default class MeasureReportBuilder<T extends PopulationGroupResult> {
         return {
           value: this.populationTotal(population, PopulationType.IPP) * 1.0
         };
-
-      case MeasureScoreType.RATIO:
-        // Note: Untested measure score type
-        // (NUMER - NUMEX) / (DENOM - DENEX)`
-        const numeratorCount2 =
-          this.populationTotal(population, PopulationType.NUMER) -
-          this.populationTotal(population, PopulationType.NUMEX);
-        const denominatorCount2 =
-          this.populationTotal(population, PopulationType.DENOM) -
-          this.populationTotal(population, PopulationType.DENEX);
-
-        return {
-          // TODO: what if value for denominator 0? ... do we need to subtract denex, dexecep... probably, as https://ecqi.healthit.gov/system/files/eCQM-Logic-and-Guidance-2018-0504.pdf
-          value: denominatorCount2 === 0 ? 0 : (numeratorCount2 / denominatorCount2) * 1.0
-        };
       default:
         throw new UnsupportedProperty(`Measure score type \"${scoringCode}\" not supported`);
     }
   }
 
-  private calculateGroupScores() {
+  public calculateGroupScores(results?: ExecutionResult<PopulationGroupResult>[]) {
```

Comment:
This function should be kept private and only use existing collected state in the report builder.